pages/linux/sslstrip.md

```diff
@@ -0,0 +1,20 @@
+# sslstrip
+
+> Perform Moxie Marlinspike's Secure Sockets Layer (SSL) stripping attacks. Users should perform ARPspoofing before using this tool.
```

Comment:
```suggestion
> Perform Moxie Marlinspike's Secure Sockets Layer (SSL) stripping attacks. Perform an ARP spoofing before using this tool.
```
- Litle typo (ARPspoofing).
- We generally use the imperative mood in the 2nd singular person instead of referring to the user in the 3rd person or using the indicative mood im the 2nd person.